JSDemos/Demos/TreeList/UsingSearchPanel/description.md

```diff
@@ -1 +1,13 @@
-This demo shows how to&nbsp;filter data by&nbsp;all columns, that&nbsp;is, to&nbsp;perform search. Type a&nbsp;search string into the search panel located in&nbsp;the top-right corner. The TreeList tries to&nbsp;find entries of&nbsp;your search string in&nbsp;the rows. If&nbsp;such entries are found, the TreeList displays the full hierarchy down to&nbsp;matching rows, but not further.
\ No newline at end of file
+The search panel allows users to filter data by all columns. To enable this feature, set **searchPanel**.[visible](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/searchPanel/#visible) to **true**. If the TreeList should not search through a specific column, set its [allowSearch](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#allowSearch) property to **false**.
+
+Search is case-insensitive. The default configuration is to search by [visible](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#visible) and invisible columns. If you want to search by visible columns only, set the **searchPanel**.[searchVisibleColumnsOnly](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/searchPanel/#searchVisibleColumnsOnly) property to **true**.
+
+Column's [data type](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#dataType) determines the condition under which the TreeList includes a row into search results:
+
+* *"number"*, *"boolean"*, and *"data"* column values should be strictly equal to the search query
+
+* *"string"* column values should contain the search query.
```

Comment:
Lists are great when you need to enumerate similar things. This is not the case. Here, we are drawing a correspondence between two things. A table could fit in this case, but there is so little information to convey. I think we can go with a simple paragraph:

```suggestion
Search is case-insensitive. Numeric, date, and Boolean values match only if they are equal to the search query. String values match even if they contain only a part of the query.
```